src/main/java/com/dumbdogdiner/stickyapi/webapis/MojangAPI.java

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) 2020 DumbDogDiner <a href="dumbdogdiner.com">&lt;dumbdogdiner.com&gt;</a>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.webapis;
+
+import com.dumbdogdiner.stickyapi.common.util.StringUtil;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Player;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.UUID;
+
+//TODO: Better error handeling in case of 404
+
+public class MojangAPI {
+
+
+    /** When possible, use the cached, faster api at https://api.ashcon.app/mojang/v2/user, otherwise use mojang
```

Comment:
Ditto

Reply:
One of these is supposed to use mojang direct, one is supposed to be ashcon, I havent split them yet.